requirements-base.txt

```diff
@@ -1,61 +1,118 @@
+# 
+# This file lists all the requirements except for the psycopg2 requirement
+# which is specified separately in requirements.txt and requirements-dev.txt
+# 
+Babel==2.8.0
+boto3==1.23.2
+botocore==1.26.2
+certifi==2021.10.8
+cffi==1.15.0
+chardet==4.0.0
+click==8.1.3
+contextlib2==21.6.0
+cryptography==37.0.2
 # defusedxml is required for SAML authentication
-defusedxml==0.6.0
-dm.xmlsec.binding==2.1
+defusedxml==0.7.1
 # docker is used for the LCP feature.
-docker==4.3.1
-docutils==0.16
+docker==5.0.3
+docutils==0.18.1
+elasticsearch==6.8.2
+elasticsearch-dsl==6.4.0
 expiringdict==1.2.1
-Flask-Cors==3.0.9
+feedparser==6.0.2
+Flask==2.1.2
+Flask-Babel==2.0.0
+Flask-Cors==3.0.10
 flask-sqlalchemy-session==1.1
 # freezegun is used for unit tests.
-freezegun==0.3.15
-html-sanitizer==1.9.1
-httplib2==0.19.0
+freezegun==1.2.1
+# fuzzywuzzy is for author name manipulations
+fuzzywuzzy==0.18.0
+html-sanitizer==1.9.3
+httplib2==0.20.4
+idna==2.10
 imagesize==1.2.0
+ipaddress==1.0.23
+isbnlib==3.10.10
+isodate==0.6.1
+itsdangerous==2.1.2
+Jinja2==3.1.2
+loggly-python-handler==1.0.1
+lxml==4.6.5
+markupsafe==2.1.1
+money==1.3.0
+multipledispatch==0.6.0
+# nameparser is for author name manipulations
+nameparser==1.1.1
 # ndg-httpsclient ensures we support SNI-based SSL
 ndg-httpsclient==0.5.1
 # newrelic is an NYPL-sepcific requirement that might be removeable.
 newrelic==5.18.0.148
+# nltk is a textblob dependency.
+nltk==3.7
 oauth2client==4.1.3
-packaging==20.4
+packaging==21.3
+Pillow==8.1.2
 pyasn1==0.4.8
 pyasn1-modules==0.2.8
+py-bcrypt==0.4
 pycparser==2.20
 pycryptodome==3.9.8
 # pyfakefs is used for unit tests.
 pyfakefs==3.7
 Pygments==2.7.4
-PyJWT==1.4.2
+PyJWT==2.4.0
 PyLD==1.0.5
+pymarc==4.0.0
+pyparsing==2.4.7
 pypostalcode==0.3.6
 pyOpenSSL==19.1.0
+pyspellchecker==0.5.5
+pytz==2021.3
+python-dateutil==2.8.2
 # TODO: python-Levenshtein is supposedly required for author name
 # matching, but is not a core requirement; perhaps it can be removed.
-python-Levenshtein==0.12.0
+python-Levenshtein==0.12.2
 # python-saml is required for SAML authentication
-python3-saml==1.10.1
+python3-saml==1.14.0
+requests==2.27.1
+requests-futures==1.0.0
 # requests-mock is used for unit tests.
 requests-mock==1.8.0
 # rsa is an oauth2client dependency. 4.3 is the last version to support Python 2.
 rsa==4.3
+s3transfer==0.5.2
+singledispatch==3.4.0.3
 snowballstemmer==2.0.0
 soupsieve==1.9.6
 # Sphinx is for code documentation
-Sphinx==1.8.5
-sphinxcontrib-websupport==1.1.2
+Sphinx==5.0.0b1
+sphinxcontrib-websupport==1.2.4
+SQLAlchemy==1.4.36
+textblob==0.15.3
 typing==3.7.4.3
 unicodecsv==0.14.1
 uritemplate==3.0.1
-uszipcode==0.2.4
+# urllib3 is pinned to a pre-1.25 version to work around https://jira.nypl.org/browse/SIMPLY-3477
+urllib3==1.26.9
+uszipcode==1.0.1
+# watchtower is for Cloudwatch logging integration
+watchtower==0.8.0
 wcag-contrast-ratio==0.9
+webpub-manifest-parser==0.0.4
```

Comment:
This package needs an [update for the path](https://github.com/NYPL-Simplified/python-webpub-manifest-parser/blob/master/src/webpub_manifest_parser/core/registry.py#L25) to `MutableMapping`. Instead of `collections.MutableMapping` it is now located at `collections.abc.MutableMapping`.

Reply:
You can fix this yourself pretty easily with a PR. However, the only developer with PyPI upload permissions is a former Lyrasis contractor who lives in Russia, so it might be tricky to contact him. 

Since we're not planning on using this for anything else, I think it'd be OK to copy this into the circulation repo and make the change there, at least as a temporary measure.